spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -98,5 +98,32 @@
       end
     end
 
+    describe "depositor counts" do
+      before do
+        GenericFile.new(id: "abc123") do |gf|
+          gf.apply_depositor_metadata user1
+          gf.update_index
+        end
+        GenericFile.new(id: "def123") do |gf|
+          gf.apply_depositor_metadata user2
+          gf.update_index
+        end
+        GenericFile.new(id: "zzz123") do |gf|
+          gf.create_date = [2.days.ago]
+          gf.apply_depositor_metadata user1
+          gf.update_index
+        end
+      end
+
+      it "gathers user deposits" do
+        get :index
+        expect(assigns[:depositors]).to include({ key: user1.user_key, deposits: 2, user: user1 }, { key: user2.user_key, deposits: 1, user: user2 })
+      end
+
+      it "gathers user deposits during a date range" do
+        get :index, deposit_stats: {start_date:1.days.ago.strftime("%Y-%m-%d"), end_date:0.days.ago.strftime("%Y-%m-%d")}
```

Comment:
Always use a single space inside braces.

Always add a space between hash keys and values (between `start_date:` and `1.days...` and between `end_date:` and `0,days...`)